lib/rgit/cli.rb

```diff
@@ -64,4 +51,50 @@ def test(*args)
       puts args
     end
   end
-end
\ No newline at end of file
+end
+
+module Impl
+  class Cli
+    def format_options(option_hash)
+      result = ""
+      option_hash.each { |k, v|
+        key_str = " --#{k.to_s.gsub("_", "-")}"
+        case v # v is truthy in all cases except: nil, false
+        when [true, false].include?(v) # it's a boolean
+          result << key_str
+        when String
+          result << key_str << "=#{v}"
+        when Fixnum
+          result << key_str << "=#{v}"
+        when Hash
+          v.each { |key, val|
+            result << key_str << " #{key}=#{val}"
+          }
+        end
+      }
+      result
+    end
+
+    def lowest_repo_above(start_dir)
+      res_dir = nil
+      start_dir.ascend { |dir|
+        if dir.has_child? ".git"
+          res_dir = dir
+          break
+        end
+      }
+      res_dir
+    end
+
```

Comment:
Unused method argument - `parent_dir`. If it's necessary, use `_` or `_parent_dir` as an argument name to indicate that it won't be used. You can also write as `init_subrepo(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `subrepo_dir`. If it's necessary, use `_` or `_subrepo_dir` as an argument name to indicate that it won't be used. You can also write as `init_subrepo(*)` if you want the method to accept any arguments but don't care about them.